.github/labeler.yml

```diff
@@ -6,20 +6,16 @@ Cython / Python:
   - 'python/**'
   - 'notebooks/**'
   
-libcuml:
+cpp:
```

Comment:
This should match an existing label. I would look at this [list](https://github.com/rapidsai/cuml/labels?sort=count-desc) and get input from @dantegd and @JohnZed on which labels they want to use and which paths to use for them